diff --git a/dist/es/components/MotionCanvas.mjs b/dist/es/components/MotionCanvas.mjs
index 083c2b4eb1609209ffa1c723c3d0161bf584bac6..87f43eddbe5fbff0e0b7edb1613c39456cd77816 100644
--- a/dist/es/components/MotionCanvas.mjs
+++ b/dist/es/components/MotionCanvas.mjs
@@ -3,7 +3,7 @@ import * as React from 'react';
 import { forwardRef, useContext, useRef, useLayoutEffect } from 'react';
 import { MotionContext, MotionConfigContext, useForceUpdate, useIsomorphicLayoutEffect } from 'framer-motion';
 import { mergeRefs } from 'react-merge-refs';
-import { createRoot, events, unmountComponentAtNode } from '@react-three/fiber';
+import { createRoot, events, unmountComponentAtNode, extend } from '@react-three/fiber';
 import { MotionCanvasContext } from './MotionCanvasContext.mjs';
 import { clamp } from 'popmotion';
 
@@ -40,6 +40,11 @@ class ErrorBoundary extends React.Component {
 ErrorBoundary.getDerivedStateFromError = () => ({ error: true });
 function CanvasComponent(_a, forwardedRef) {
     var { children, fallback, tabIndex, id, style, className, events: events$1 } = _a, props = __rest(_a, ["children", "fallback", "tabIndex", "id", "style", "className", "events"]);
+    
+    // Create a known catalogue of Threejs-native elements
+  // This will include the entire THREE namespace by default, users can extend
+  // their own elements by using the createRoot API instead
+  React.useMemo(() => extend(THREE), [])
     /**
      * Import existing contexts to pass through variants and MotionConfig from
      * the DOM to the 3D tree. Shared variants aren't officially supported yet